newsfragments/validate_files.py

```diff
@@ -8,6 +8,7 @@
 import sys
 
 ALLOWED_EXTENSIONS = {
+    '.breaking-change.rst',
```

Comment:
To match all the other one-word types, maybe `s/breaking-change/breaking/g`? It's a little shorter, and doesn't lose too much meaning, I think.

Reply:
I like that! Will update!